docs/l10n.rst

```diff
@@ -502,6 +502,16 @@ You get an alphabetically sorted list of all of the active locales for that .ftl
 You should run ``./manage.py l10n_update`` as shown above for the most accurate and
 up-to-date results.
 
+String extraction
+=================
+
+The string extraction process for both new .ftl content and updates to existing .ftl
+content is handled through automation. On each commit to master a command is run that
+looks for changes to the ``l10n/`` directory. If a change is detected, it will copy
+those files into a new branch in ``git-repos/l10n-team`` and then a bot will open a
```

Comment:
Technically it's true that it's the same repo as `git-repos/l10n-team`, but the branch is created in https://github.com/mozilla-l10n/www-l10n, so maybe linking to that repo is the best thing here?